buildpack/telemetry/splunk.py

```diff
@@ -0,0 +1,41 @@
+import logging
```

Comment:
For the sake of consistency, let's add a `def run()` method in splunk.py. And it could simply log that Splunk has been successfully enabled.

Reply:
added function "ready", which prints log line